css/fractalCss/card.css

```diff
@@ -0,0 +1,135 @@
+/* Cards */
+.card {
+  display: flex;
+  width: 100%;
+}
+
+.card--verticle {
+  flex-direction: column;
+  justify-content: center;
+  align-items: center;
+}
+
```

Comment:
Here,  there is a replication of `justify-content` and `align-items` they can be added in the `card` class itself and flex sets the direction to `row` by default. So, it can be removed from `card--horizontal`